xbmc/guilib/GUIMoverControl.cpp

```diff
@@ -147,14 +148,14 @@ void CGUIMoverControl::OnRight()
 
 EVENT_RESULT CGUIMoverControl::OnMouseEvent(const CPoint &point, const CMouseEvent &event)
 {
-  if (event.m_id == ACTION_MOUSE_DRAG)
+  if (event.m_id == ACTION_MOUSE_DRAG || event.m_id == ACTION_MOUSE_DRAG_END)
```

Comment:
Optional, but you can also replace `ACTION_*` ints with an enum class. Keep the int values with each enum label.

Reply:
on ActionIDs.h there are about 400 defined ACTION_*
change all them with an enum class is needed change a lot of code
is that what you mean?

if so, it seems superfluous to me to keep the current assigned fixed numbers of all ACTION_*, why not use enum class automatic values?
however i think better do it on another PR

Reply:
If it's a lot of code then yes another PR. And it goes without mentioning that it'd be appreciated. Not automatic enum values because action IDs are exposed to Python, and they have int values there, therefore this must be the source of truth.

Reply:
are you know if has been done something similar? so change all constexpr with an enum class to be exposed in python?
i wonder if current python interface support expose enum class

Reply:
I made a mistake, our ACTION_IDs are directly parsed by swig which probably can't handle enum classes. Disregard this thread.

Reply:
just as a comment, its possible, just we dont do it currently